backend/directus/events.go

```diff
@@ -0,0 +1,101 @@
+package directus
+
+import (
+	"github.com/bcc-code/mediabank-bridge/log"
+	"github.com/gin-gonic/gin"
+	"github.com/samber/lo"
+	"strconv"
+)
+
+type Event struct {
+	Event          string        `json:"event"`
+	Accountability any           `json:"accountability"`
+	Payload        any           `json:"payload"`
+	Collection     string        `json:"collection"`
+	Keys           []interface{} `json:"keys"`
+	Key            int           `json:"key"`
+}
+
+const (
+	EventItemsCreate = "items.create"
+	EventItemsUpdate = "items.update"
+	EventItemsDelete = "items.delete"
+)
+
+type EventHandler struct {
+}
+
+func getModelFromCollectionName(collection string) string {
+	switch collection {
+	case "episode", "episodes":
+		return "episode"
+	case "season", "seasons":
+		return "season"
+	case "show", "shows":
+		return "show"
+	}
+	return ""
+}
+
+var itemsEvents = map[string][]func(model string, id int){}
+
+func (handler *EventHandler) On(events []string, callback func(model string, id int)) {
+	for _, event := range events {
+		switch event {
+		case EventItemsUpdate, EventItemsCreate, EventItemsDelete:
+			log.L.Debug().Msgf("Registering Directus webhook-listener for event: \"%s\"", event)
```

Comment:
```suggestion
			log.L.Debug().Str("event", event).Msg("Registering Directus webhook-listener")
```
This way we can get event as its own field and can use it for filtering etc later.

Same on line 60 for example